framework/test/jest/specs/integration/in_memory_channel.spec.js

```diff
@@ -0,0 +1,200 @@
+const InMemoryChannel = require('../../../../src/controller/channels/in_memory_channel');
+const Bus = require('../../../../src/controller/bus');
+const Event = require('../../../../src/controller/event');
+
+const logger = {
+	info: jest.fn(),
+};
+
+const config = {
+	ipc: {
+		enabled: false,
+	},
+};
+
+const alpha = {
+	moduleAlias: 'alphaAlias',
+	events: ['alpha1', 'alpha2'].map(
+		event => new Event(`${'alphaAlias'}:${event}`)
+	),
+	actions: {
+		multiplyByTwo: action => action.params * 2,
+		multiplyByThree: action => action.params * 3,
+	},
+};
+
+const beta = {
```

Comment:
can we put these constants into the scope of describe?

Reply:
@shuse2 actually I didn't have a strong opinion against this but now I kind of have :D 

Moving variables into describe block, increases the `complexity` score of describe blocks. I think intuitively having them outside of describe tells us that they are not being mutated by test hooks. I prefer to keep only the variables that we are mutating inside the describe block.

@lsilvs @limiaspasdaniel what do you guys think about this?

For instance when 
![image](https://user-images.githubusercontent.com/26697004/55010587-bcc9f280-4fe4-11e9-8a32-4d79a1eeba7d.png)

Reply:
I have no strong opinion on this. LGTM.